### Calculating Probabilities

In [2]:
import numpy as np
import matplotlib.pyplot as plt
from scipy import stats

In [3]:
x = 75
mean = 60
std_dev = 40
samp_size = 10
standard_error = std_dev / (samp_size**.5)

stats.norm.cdf(x, mean, standard_error)



0.8821600432854813

The probability that the average weight of our sample of 10 fish will be less than or equal to 75 lbs is 88.2%.